Watch Extension/UnidirectionalFlow.swift

```diff
@@ -36,22 +44,31 @@ final class Store<Value, Action>: ObservableObject {
 }
 
 func combine<Value, Action>(
-    _ reducers: (inout Value, Action) -> Void...
-) -> (inout Value, Action) -> Void {
+    _ reducers: Reducer<Value, Action>...
+) -> Reducer<Value, Action> {
     return { value, action in
-        for reducer in reducers {
-            reducer(&value, action)
-        }
+        let effects = reducers.flatMap { $0(&value, action) }
+        return effects
     }
 }
 
 func pullback<GlobalValue, LocalValue, GlobalAction, LocalAction>(
-    _ reducer: @escaping (inout LocalValue, LocalAction) -> Void,
+    _ reducer: @escaping Reducer<LocalValue, LocalAction>,
     value: WritableKeyPath<GlobalValue, LocalValue>,
     action: WritableKeyPath<GlobalAction, LocalAction?>
-) -> (inout GlobalValue, GlobalAction) -> Void {
+) -> Reducer<GlobalValue, GlobalAction> {
     return { globalValue, globalAction in
-        guard let localAction = globalAction[keyPath: action] else { return }
-        reducer(&globalValue[keyPath: value], localAction)
+        guard let localAction = globalAction[keyPath: action] else { return [] }
+        let localEffects = reducer(&globalValue[keyPath: value], localAction)
+        let globalEffects: [Effect<GlobalAction>] = localEffects.map { localEffect in
+            { callback in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/995ebb3b78eb3cd87dba718e07e1f295f49795f1/Watch Extension/UnidirectionalFlow.swift#L64"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>UnidirectionalFlow.swift:64</code></td>
    </tr>
  </tbody>
</table>